docs/_tutorials/getting_started.md

```diff
@@ -176,7 +176,7 @@ const app = new App({
 // Listens to incoming messages that contain "hello"
 app.message('hello', ({ message, say }) => {
   // say() sends a message to the channel where the event was triggered
-  say(`Hey there <@{message.user}!`);
+  say(`Hey there ${message.user}!`);
```

Comment:
```suggestion
  say(`Hey there <@${message.user}>!`);
```

I think you're both right 😆. The Slack mention syntax is meant to be surrounded by the JavaScript template literal placeholder syntax.